requirements-dev.txt

```diff
@@ -1,9 +1,19 @@
+alabaster
 flake8
+flake8-builtins
+flake8-comprehensions
+flake8-import-order
+flake8-mutable
+flake8-print
+flake8-quotes
 mock
+nbsphinx
```

Comment:
With this extension we can convert Jupyter notebooks examples to docs.

Reply:
Fantastic! I like the example from the folium docs